src/components/PDFView/index.js

```diff
@@ -160,6 +161,21 @@ class PDFView extends Component {
             </View>
         );
     }
+
+    render() {
+        return this.props.onPress ? (
+            <PressableWithoutFeedback
+                onPress={this.props.onPress}
+                style={[styles.flex1, styles.flexRow, styles.alignSelfStretch]}
```

Comment:
adding `styles.flexRow` caused a regression where failed to load message is not properly visible on native https://github.com/Expensify/App/pull/21714